app/src/parser_impl.h

```diff
@@ -38,9 +38,11 @@ parser_error_t _validateTx(const parser_context_t *c, const parser_tx_t *v);
 
 uint8_t _getNumItems(const parser_context_t *c, const parser_tx_t *v);
 
-uint8_t _countArgumentItems(const flow_argument_list_t *v, uint8_t argumentIndex);
+parser_error_t _countArgumentItems(const flow_argument_list_t *v, uint8_t argumentIndex, 
```

Comment:
I know the code was already missing some documentation here, but it would be nice to add a brief description of what the parameters represent. This would help the code maintainability in the future.